### PR TITLE
Update lan_admin.php

### PR DESCRIPTION
**Pull request by yesszus**

### CODE REVIEW

**Comment by yesszus** [comment]
@CaMer0n  pls review this.

File: e107_admin/admin.php [hunk 1]
```diff
@@ -535,7 +535,7 @@ private function checkPasswordEncryption()
 
 		if($us->passwordAPIExists() === true && $us->getDefaultHashType() !== PASSWORD_E107_PHP && e107::pref('core','password_CHAP')==0)
 		{
-			$message = "It is HIGHLY recommended that you [change your password encoding] to the PHP Default. (Password hashes will be automatically upgraded during user login.)";
+			$message = e107::getParser()->toHTML(LAN_UI_PASSWORD_WARNING, true);
```
Comment (CaMer0n) [inline e107_admin/admin.php]: This is not part of the admin-ui, so LAN_UI is not an appropriate prefix. 
Also, it does not contain regular bbcodes such as `[b]`,  so running it through the parser is not needed.

Comment (yesszus) [inline e107_admin/admin.php]: Ok @CaMer0n changed this line and removed regular bbcode.

**Comment by CaMer0n** [comment]
Thank you! 👍

**Pull request merged**